proto/wippersnapper/pwm.proto

```diff
@@ -3,6 +3,26 @@
 syntax = "proto3";
 package wippersnapper.pwm;
 
+/**
+ * BrokerToDevice message envelope
+ */
+message PWMB2D {
+  oneof payload {
+    PWMAdd pwm_add       = 10;
+    PWMRemove pwm_remove = 11;
+    PWMWrite pwm_write   = 12;
+  }
+}
+
+/**
+ * DeviceToBroker message envelope
+ */
+message PWMD2B {
+  oneof payload {
+    PWMAdded pwm_added = 10;
+  }
```

Comment:
@lorennorman  For `message`s like this, where there's only one `payload` value, would we still want to use the `oneof`, or would we reference the message? I see this across the proto files.